### ISSUE TITLE
Mulitple model submission --overwrite flag is not passed through

### ISSUE THREAD

**Issue by seth127** [metrumresearchgroup/babylon#156]
What seems to be happening is that, when you pass multiple model files as arguments, the `--overwrite` CLI flag is only passed through to the _first_ of those models.

Example:
```
$ cp 1.ctl 2.ctl

$ /data/apps/bbi nonmem run sge 1.ctl 2.ctl

$ /data/apps/bbi nonmem run sge 1.ctl --overwrite
INFO[0000] Successfully loaded default configuration from /data/home/sethg/rbabylon/tests/testthat/model-examples-bbi/babylon.yaml
INFO[0000] Beginning Local Path
INFO[0000] A total of 1 models have been located for work
INFO[0000] [1] Beginning SGE work phase
1 models completed in 10.174348ms

$ /data/apps/bbi nonmem run sge 1.ctl 2.ctl --overwrite
INFO[0000] Successfully loaded default configuration from /data/home/sethg/rbabylon/tests/testthat/model-examples-bbi/babylon.yaml
INFO[0000] Beginning Local Path
INFO[0000] A total of 2 models have been located for work
INFO[0000] [1] Beginning SGE work phase
ERRO[0000] 1 errors were experienced during the run
ERRO[0000] Errors were experienced while running model 2. Details are The target directory, /data/home/sethg/rbabylon/tests/testthat/model-examples-bbi/2 already exist, but we are configured not to overwrite. Invalid configuration / run state
2 models completed in 10.180546ms
```

I originally thought maybe _none_ of the flags were passed through, but I tried it with `threads=4` and then looked at `1/bbi_config.json` and `2/bbi_config.json` and they both had `threads: 4` in them. (When I didn't pass the flag, they both had `threads: 2` in them.

**Comment by shairozan** [comment]
This isn't actually an issue with it not being passed through, but rather an issue of concurrency. We use a pointer to the configuration object as a child to the Nonmem objects, so basically they're all _sharing_ a configuration as opposed to copying it. 

What is happening, however, is that the values are manipulated during SGE submission such that the values of the babylon.yaml file written into the output dir are modified. Three things are changed:

* Output = false
* CreateChildDirs = false
* SaveConfig = false

Because we're dealing with a pointer to the configuration here, in a timed scenario, model 1 could finish and try to save its yaml (setting the pointer value to false) so when the next model goes to run it indicates that it's false. I also verified that it would only periodically happen. IE sometimes I could submit SGE fine 10 times in a row without issue, but would never happen with local execution. 

Fix should be simple. In the block where we change the values and write to file, we will dereference and create a local copy that we will manipulate before writing to file.

**Comment by seth127** [comment]
Update: https://github.com/metrumresearchgroup/babylon/pull/158 fixed this issue.

@shairozan provided a binary to me with the fix from the PR mentioned above and it fixed the issue described here. Thanks!

**Comment by shairozan** [comment]
Fix is now available in release 2.1.1-rc1
